org.eclipse.winery.repository.rest/src/test/java/org/eclipse/winery/repository/rest/resources/entitytypeimplementations/nodetypeimplementations/NodeTypeImplementationResourceTest.java

```diff
@@ -17,7 +17,27 @@
 import org.junit.Test;
 
 public class NodeTypeImplementationResourceTest extends AbstractResourceTest {
+	private final String ENTITY_TYPE = "nodetypeimplementations/";
+	private final String INSTANCE_XML_PATH = "entityimplementations/" + ENTITY_TYPE + "instance.xml";
+	private final String BAOBAB_JSON_PATH = "entityimplementations/" + ENTITY_TYPE + "baobab_inital.json";
```

Comment:
I really would inline these lines to have the tests maintainable. The test classes will grow large and these files will be a small subset.

Reply:
If you decide for anything else, please use the ADR template available at https://github.com/OpenTOSCA/winery/blob/master/docs/adr/template.md.

Reply:
After reading the content of the files, the filename does not reflect the content. (See below). Please rename the file according to its content and adapt the test cases accordingly.